fdbserver/TLogServer.actor.cpp

```diff
@@ -854,7 +818,7 @@ void commitMessages( Reference<LogData> self, Version version, const std::vector
 	self->messageBlocks.push_back( std::make_pair(version, block) );
 	addedBytes += int64_t(block.size()) * SERVER_KNOBS->TLOG_MESSAGE_BLOCK_OVERHEAD_FACTOR;
 
-	self->version_sizes[version] = make_pair(expectedBytes, expectedBytes);
+	self->version_sizes[version] = std::make_pair(expectedBytes, txsBytes);
```

Comment:
I did a quick look and didn't see where we were using the different version sizes distinctly (txsBytes vs. non-txsBytes). Can this just be combined to one value?

Reply:
I missed the one spot where we did use the first value only. We can disregard this comment.